lib/member.symphony.php

```diff
@@ -413,13 +413,15 @@ public function filter_ValidatePassword(array &$context) {
 				$password = $_POST['fields'][$this->section->getFieldHandle('authentication')]['validate'];
 
 				if ($password) {
-					// Handle which is the Identity field, either the Member: Username or Member: Email field
-					$identity = is_null($this->section->getFieldHandle('identity')) ? 'email' : 'identity';
+					// Handle which is the Identity field, either the Member: Username or Member: Email field;
+					// the data array passed to findMemberIDFromCredentials() needs either 'username' or
+					// 'email' as key, so we must not use the string 'identity' in this case.
+					$identity = is_null($this->section->getFieldHandle('identity')) ? 'email' : 'username';
```

Comment:
Would ! be better ?

Reply:
This is adopted from here: https://github.com/symphonycms/members/blob/1.10.1/lib/member.symphony.php#L417. I wouldn't like to do it differently when re-using it.

Reply:
Ok great.